Content.Shared/GameObjects/Components/Destructible/DamageVisualizerData.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Content.Shared.GameObjects.Components.Destructible
+{
+    public enum DamageVisualizerData
```

Comment:
byte

Reply:
Fixed in https://github.com/space-wizards/space-station-14/pull/2783/commits/4a91bbdc7afcdce7c2b04947116626ca87c9a0a6